insanely_fast_whisper_rocm/core/integrations/diarization.py

```
@@ -0,0 +1,472 @@
+            )
+            return audio_path
+
+        waveform, sample_rate = torchaudio.load(tmp_path)  # type: ignore[possibly-undefined]
+        logger.debug(
+            "Preloaded audio via ffmpeg→torchaudio: shape=%s, sr=%d, src=%s",
+            waveform.shape,
+            sample_rate,
+            suffix,
+        )
+        return {"waveform": waveform, "sample_rate": sample_rate}
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)